config/config.go

```diff
@@ -165,6 +165,7 @@ type AgentConfig struct {
 	// TODO(cam): Remove UTC and parameter, they are no longer
 	// valid for the agent config. Leaving them here for now for backwards-
 	// compatibility
+	// Deprecated: 1.0.0 after, has no effect
```

Comment:
Looks like this change duplicates a comment already there, was this change intentional?

Reply:
adjust the deprecated position, some IDE can disply strikethrough, eg: goland
effect：
![image](https://user-images.githubusercontent.com/3955934/142526867-3777ec2b-eb81-494e-916f-97289b2d4b6c.png)

Reply:
Ok cool I didn't realize Golang had that feature, can you then remove the old comment that is identical?

Reply:
ok